easybuild/easyconfigs/b/bigdft-suite/bigdft-suite-1.9.1-foss-2021b.eb

```
@@ -0,0 +1,109 @@
+easyblock = "Bundle"
+
+name = "bigdft-suite"
+version = "1.9.1"
+local_version = "1.9.1"
+
+homepage = "https://gitlab.com/l_sim/bigdft-suite"
+description = """
+BigDFT: electronic structure calculation based on Daubechies wavelets.
+bigdft-suite is a set of different packages to run bigdft.
+"""
+
+toolchain = {"name": "foss", "version": "2021b"}
+toolchainopts = {
+    "usempi": True,
+    "extra_fflags": "-fallow-argument-mismatch",
+    "extra_fcflags": "-fallow-argument-mismatch",
+    "extra_f90flags": "-fallow-argument-mismatch -lgain",
+}
+
+default_easyblock = "ConfigureMake"
+default_component_specs = {
+    "source_urls": [
+        "https://gitlab.com/l_sim/bigdft-suite/-/archive/%s/" % local_version
+    ],
+    "sources": ["bigdft-suite-%s.tar.gz" % local_version],
+    "start_dir": "bigdft-suite-%s" % local_version,
+}
+
+
+local_configopts = """CC=\\"$MPICC\\" FC=\\" $MPIFC \\" \
+--enable-mpi \
+\
+FCFLAGS=\\"$CPPFLAGS -fallow-argument-mismatch -I%(installdir)s/include \
+-I$EBROOTSCALAPACK/include $FCFLAGS -std=legacy \\" \
+\
+FFLAGS=\\"$CPPFLAGS -fallow-argument-mismatch -I%(installdir)s/include \
+-I$EBROOTSCALAPACK/include $FFLAGS -std=legacy \\" \
+\
+F90FLAGS=\\"$CPPFLAGS -fallow-argument-mismatch -I%(installdir)s/include \
+-I$EBROOTSCALAPACK/include $F90FLAGS -std=legacy \\" \
+\
+--with-ext-linalg=\\"-L$SCALAPACK_LIB_DIR $LIBSCALAPACK -L$LAPACK_LIB_DIR $LIBBLAS \\" \
+--with-yaml-path=\\"$EBROOTLIBYAML\\" """
```

for readability, consider something like:
```suggestion
local_configopts = ' '.join([
    'CC="$MPICC" FC="$MPIFC" --enable-mpi',
    'FCFLAGS="$CPPFLAGS $FCFLAGS -std=legacy"',
    'FFLAGS="$CPPFLAGS $FFLAGS -std=legacy"',
    'F90FLAGS="$CPPFLAGS $F90FLAGS -std=legacy"',
    '--with-ext-linalg="-L$SCALAPACK_LIB_DIR $LIBSCALAPACK -L$LAPACK_LIB_DIR $LIBBLAS"',
    '--with-yaml-path="$EBROOTLIBYAML"',
])
```

I've stripped the `-fallow-argument-mismatch` duplicated from the `toolchainopts`, and removed `-I$EBROOTSCALAPACK/include` (should already be in `$CPPFLAGS`, and `-I%(installdir)s/include` (which should be empty prior to the install step)

---

adopted, thank you